test/AlphaGovernor.t.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.20;
+
+import {console2} from "forge-std/console2.sol";
+
+import {IERC20} from "@openzeppelin/contracts/interfaces/IERC20.sol";
+import {ERC20VotesComp} from "@openzeppelin/contracts/token/ERC20/extensions/ERC20VotesComp.sol";
+import {IGovernor} from "@openzeppelin/contracts/governance/IGovernor.sol";
+import {IGovernorAlpha} from "src/interfaces/IGovernorAlpha.sol";
+import {RadworksGovernorTest} from "test/helpers/RadworksGovernorTest.sol";
+import {ProposalTest} from "test/helpers/ProposalTest.sol";
+
+abstract contract AlphaGovernorTest is ProposalTest {
```

Comment:
Nit: Would `GovernorAlpha` be a more correct name?

Reply:
Or maybe `RadworksGovernorAlpha`

Reply:
Done 4acddf56dc8c1f3ff50a91e8783d74033e7883eb